pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
```

Comment:
You are using an older version of pnpm, consider running `corepack enable` and then install again.

Reply:
Thanks! I think I've fixed it... 👀